src/manifestParser/manifestV3.ts

```diff
@@ -70,13 +74,31 @@ export default class ManifestV3 extends ManifestParser<Manifest> {
     return result;
   }
 
+  protected parseInputWebAccessibleScripts(
+    result: ParseResult<Manifest>
+  ): ParseResult<Manifest> {
+    result.manifest.web_accessible_resources?.forEach((struct) => {
+      struct.resources.forEach((scriptFile) => {
```

Comment:
Same comment here re: need to limit to handling resources that match the filter and single files